qiskit/test/base.py

```diff
@@ -62,6 +62,7 @@ class BaseTestCase(testtools.TestCase):
         assertRaises = unittest.TestCase.assertRaises
         assertEqual = unittest.TestCase.assertEqual
 
+
 else:
```

Comment:
I think you might need to update your version of black - I think this line got removed in the latest bump.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7374/commits/82d3576f13866eb006314ce338da7ef54b0053d6